Quadriga/src/main/java/edu/asu/spring/quadriga/service/network/impl/NetworkXMLParser.java

```diff
@@ -0,0 +1,112 @@
+package edu.asu.spring.quadriga.service.network.impl;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.io.StringWriter;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.TransformerFactoryConfigurationError;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+import edu.asu.spring.quadriga.domain.factory.networks.INetworkXMLFactory;
+import edu.asu.spring.quadriga.domain.impl.workspace.TextFile;
+import edu.asu.spring.quadriga.domain.network.INetworkXML;
+import edu.asu.spring.quadriga.exceptions.NetworkXMLParseException;
+import edu.asu.spring.quadriga.service.network.INetworkXMLParser;
+
+/**
+ * 
+ * @author skollur1
+ *
+ */
+@Service
+public class NetworkXMLParser implements INetworkXMLParser {
+
+    @Autowired
+    private INetworkXMLFactory nwXMLFactory;
+
+    private INetworkXML networkXML;
```

Comment:
Let's talk about stateful and stateless classes again. This class variable would cause huge issues.